### PR TITLE
Completed cards and result page

### PR DESCRIPTION
**Pull request by aayushjain-code**

### CODE REVIEW

**Comment by aayushjain-code** [comment]
https://628c87ff6e91c146c38c388c--dashing-belekoy-848917.netlify.app

File: src/Components/Questions/Question.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	margin-left: 35rem;
+}
+.container-question {
+	border: 1px solid var(--text-color-3);
+	border-radius: 1rem;
+	width:80rem;
+	padding: 2rem 4rem;
+}
+.button-controller {
+	padding: 1rem 1rem !important;
```
Comment (MuskanChhatrasal) [inline src/Components/Questions/Question.css:13]: Try to avoid using important keyword

File: src/Components/Rules/Rules.css [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
-	align-items: center;
-	text-align: center;
+      margin-top: 15rem;
+      align-items: center;
+      text-align: center;
+}
+
+
+.component-display-container {
+      border: none !important;
```
Comment (MuskanChhatrasal) [inline src/Components/Rules/Rules.css:9]: Here also

File: src/Components/Rules/Rules.jsx [hunk 3]
```diff
@@ -44,13 +44,9 @@ export const Rules = () => {
             <div class="grid-3-column-layout">
```
Comment (MuskanChhatrasal) [inline src/Components/Rules/Rules.jsx:44]: Use className instead of class here

**Review by MuskanChhatrasal — APPROVED** [review]
Amazing UI with readable code. Just suggested few things try to implement those else everything looks good to merge.

**Review by mikesch803 — APPROVED** [review]
great work! you can merge this

**Pull request merged**